docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -745,8 +745,9 @@ https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql?view=sq
 If any of the two arguments is `null` or the pattern is an empty string `null` is returned.
 
 [NOTE]
-If the 1st argument is of type `time`, then pattern specified by the 2nd argument cannot contain date related units
-(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned.
+[1] If the 1st argument is of type `time`, then pattern specified by the 2nd argument cannot contain date related units
+(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned. +
+[2] Format specifier `F` will be working similar to format specifier `f`.
```

Comment:
I'd rather have it as a separate `[NOTE]` and please add more detail: that it doesn't work as optional (0 digits will be printed out).

Reply:
Added separate `[NOTE]` and added more details about `F` format specifier.